index.js

```diff
@@ -64,25 +47,7 @@ module.exports = function ({ content, filename, sourceMap }) {
       + transpile(`${exportsTarget}.staticRenderFns = [${staticRenderFns}];`)
       + `\n${exportsTarget}.render._withStripped = true;`;
   }
-  const result = `${scriptPart}${compiledTemplate}`;
```

Comment:
You mean that parseComponent will add lines to fill original `styles` and `template` part?

If there is no problem while consuming break point debugging, just merge.